projects/convex-hull/index.md

```diff
@@ -42,8 +42,16 @@ problem and the different algorithms to solve it.
 
 ## Testing
 
-Verify that the actual output matches the expected output. See the
-[requirements][3] section for an example of the expected output.
+The following table contains various test cases that you can use to
+verify the correctness of your solution; please note that different algorithms
+could produce the output starting from a different point, and/or in the opposite direction:
+
+| Description         | Input X | Input Y | Output |
+|---------------------|---------|---------|--------|
+| X-Ordered triangle  | 100, 180, 240                         | 220, 120, 20                         | (100, 220)<br>(240, 20)<br>(180, 120)                            |
```

Comment:
Can we add some error cases? Like for instance, x and y are different sizes. And, y is missing.

Reply:
Done